docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -76,6 +106,13 @@ $ cd opt/alluxio*
 $ bin/alluxio runTests
 ```
 
+### Sharing ramdisk with clients
+
+Running the worker with `-v /mnt/ramdisk:/mnt/ramdisk` will make the worker's ramdisk
```

Comment:
the point I'm making here is that the data written by the worker is available on the host. I've updated the wording to be neutral about which way the sharing is happening